subprojects/core-api/src/main/java/org/gradle/api/artifacts/MutableVersionConstraint.java

```diff
@@ -48,6 +48,7 @@
     /**
      * Sets the required version of this module. Any other version constraints will be overriden.
      * @param version the preferred version of this module
+     * @since 4.11
```

Comment:
Needs to be `4.10`

Reply:
Are you sure? Assuming this PR is merged into `master`, then the first version this method will appear is `4.11` (or 5.0, I guess).

Reply:
I am wrong indeed, had in mind this was going to `release` when we explicitly said it was not, sorry.